src/services/ActivityCardService.js

```diff
@@ -1 +1,146 @@
-// placeholder file for Drive shenanigans woohoo!
+import {
+  checkFileExists,
+  writeFile,
+  makeDirectory,
+  moveFile,
+  deleteFile,
+} from './routes/Local';
+import { MAINDIRECTORY } from './constants';
+import { AccessToken } from './models';
+import axios from 'axios';
+import { DRIVE_API_URLS } from '../config.json';
+
+const ActivityCardService = {
+  // All APIs for ActivityCards should be here
+  // Don't call RNFS directly here!
+  // Instead, call a function from Local.js :~]
+
+  // const auth = await google.auth.getClient({
+  //   credentials: YOUR_AUTH_CREDENTIALS,
+  //   scopes: ['https://www.googleapis.com/auth/drive'],
+  // });
+
+  /**
+   * Returns array of file paths for this month's featured activity cards.
+   * @return {String[]}
+   */
+  getFeaturedActivityCards: async function () {
+    try {
+      //Retrieve current year and month from browser
+      const today = new Date();
+      var currYear = String(today.getFullYear());
+      var currMonth = String(today.getMonth() + 1).padStart(2, '0'); //Account for January being '0'
+
+      //Set up GET url and path to Featured Card directory
+      var downloadUrl =
+        DRIVE_API_URLS.SEARCH_FILES +
+        'supportsAllDrives=true&trashed=false&includeItemsFromAllDrives=true&fields=files(id,kind,driveId,mimeType,teamDriveId,name,createdTime)';
```

Comment:
if you're always using these call params, maybe move this into constants.js and call it from there?

Reply:
Ye or `config.json`

Reply:
Looking at the `downloadUrl`, this is a lot of files to get and parse through. Ideally, Drive does most of the work for us. I would look into adding a query for [Image or video files modified after a specific date](https://developers.google.com/drive/api/guides/search-files#custom) into the url parameters. However, I would try and avoid looping over a lot of Drive calls, so be smart about what date you're choosing to put in this q.